proposals/0317-sdl-protocol-security-specification.md

```diff
@@ -17,60 +17,75 @@ SDL has used protected services since the start of mobile navigation apps. Unfor
 <table>
 
 ### Change 2: Add New Section
 
 The following section should be added to the protocol spec:
 
-## 4.7 Secured Communication
-
-It is possible to establish a secured and encrypted communication with the system by setting the frame header encryption flag to `1` when starting a new service. If the authentication was successful, the system will reply with a `StartService ACK` frame with the encryption flag also set to `1` indicating that encrypted data is now accepted. If the authentication fails for some reason the system will reset the TLS connection and return a `StartService NAK` frame.
-
-### 4.7.1 Authentication
+## 5.1.1 Security Query
 
-The below diagram shows the sequence of how the TLS handshake exchanges certificates to compute the master secret.
-
-![TLS Handshake activity diagram](../assets/proposals/0317-sdl-protocol-security-specification/tls-handshake.png)
-
-The authentication is done using TLS handshake. The TLS handshake process is defined by TLS and is not part of the SDL protocol. The handshake is designed as a client server communication which is configurable in the system settings. An application can take the role of a server where the system is the client or vice versa. This setting is not dynamic which means an SDL integrator must agree on one setup and avoid Client/Client or Server/Server connections. The client entity will initiate a TLS handshake with the corresponding security manager of the server. The client will do this only if the server was not authenticated before in the current transport connection. According to the TLS handshake process the peer certificate can be omitted for the server but it's required for the client.
-
-The system can be configured to support one encryption method. The following methods are supported:
-
-- SSLv3
-- TLSv1
-- TLSv1.1
-- TLSv1.2
-- DTLSv1
+The TLS Payload is sent in a binary query header. The size is 12 bytes which matches the size of the RPC Payload Binary Header.
 
-Depending on the role, the system has to initiate with the corresponding server method or client method.  For instance, if the system is configured to use DTLSv1 and its role is set to client, the system has to use the method DTLSv1_client. The application role has to be server and must use DTLSv1_server.
+### 5.1.1.1 Payload
 
-### 4.7.1 Security Query
+The security query is able to contain JSON data as well as binary data. During the handshake the TLS handshake data is sent as binary data. In case of an error, a notification is sent with an error code and error message as JSON data. See "Error handling" section for details.
 
-The TLS Payload is sent in a binary header. The size is 12 bytes which matches the size of the RPC Payload Binary Header.
+<table width="100%">
+  <tr><td align="center">Binary Query Header</td></tr>
+  <tr><td align="center">JSON Data</td></tr>
+  <tr><td align="center">Binary Data</td></tr>
+</table>
 
-#### 4.7.1.1 Binary Query Header
+### 5.1.1.2 Binary Query Header
```

Comment:
```suggestion
### 5.1.1.2 Binary Header
```
See Item 4 within [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/1070#issuecomment-724282962) outlining Steering Committee requested revisions.

Reply:
@theresalech fixed in https://github.com/smartdevicelink/sdl_evolution/pull/1138/commits/2a6995a428cb531a3f50bfd8253e96dcffc3548f